opm/simulators/flow/SimulatorFullyImplicitBlackoilEbos.hpp

```diff
@@ -241,7 +241,7 @@ class SimulatorFullyImplicitBlackoilEbos
                 path.replace_filename(ioconfig.getBaseName() + ".OPMRST");
                 loadFile_ = path;
                 if (!std::filesystem::exists(loadFile_)) {
-                    OPM_THROW(std::runtime_error, "Error locating serialized restart file");
+                    OPM_THROW(std::runtime_error, "Error locating serialized restart file " + saveFile_);
```

Comment:
Should be loadFile_ I think? Otherwise a good idea.

Reply:
Indeed. fixed.